Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "kwctl"
 description = "Tool to manage Kubewarden policies"
-version = "1.0.1"
+version = "1.0.2"
```

Comment:
this means a major version bump for the rest of the components and charts.  I suppose we are happy with having this PR land in main, and then make the new tags of kwctl, policy-server (for the helm chart update), helm charts.